editoast/src/views/path/properties.rs

```diff
@@ -175,15 +237,53 @@ async fn post(
 ) -> Result<Json<PathProperties>> {
     // Extract information from parameters
     let conn = &mut db_pool.get().await?;
-    let infra_version = retrieve_infra_version(conn, infra_id).await?;
     let query_props: Properties = props.into();
     let mut valkey_conn = valkey.get_connection().await?;
+    let infra = Infra::retrieve_or_fail(conn, infra_id, || PathfindingError::InfraNotFound {
+        infra_id,
+    })
+    .await?;
+
+    // Get track section offsets.
+    let track_section_offsets = path_properties_input.find_track_section_offset();
+
+    // Get the IDs of the track sections.
+    let object_ids = path_properties_input
+        .find_track_section_offset()
```

Comment:
Let's reuse the variable just above.

```suggestion
    let object_ids = track_section_offsets
```

Reply:
Done.